packages/admin-panel/src/VizBuilderApp/components/PreviewOptions/LocationField.jsx

```diff
@@ -7,6 +7,7 @@ import React, { useState } from 'react';
 import { Autocomplete } from './Autocomplete';
 import { useVizConfigContext } from '../../context';
 import { useEntityByCode, useLocations } from '../../api';
+import { EntityOptionLabel } from '../../../widgets/EntityOptionLabel';
```

Comment:
This is very minor, but to keep with our pattern, can we export `EntityOptionLabel` from `widgets/index.js` and then this can just be imported from the `widgets` folder?